Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -91,14 +91,19 @@ public class SeleniumDriver : DriverBase, IVirtualDriver, IWindowExplorer, IVisu
         private const string EDGE_DRIVER_NAME = "msedgedriver";
         private const string FIREFOX_DRIVER_NAME = "geckodriver";
         private const string TRANSLATOR_FOR_CASE_INSENSITIVE_MATCH = "translate(text(), 'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 'abcdefghijklmnopqrstuvwxyz')";
+        private const string BRAVE_32BIT_BINARY_PATH = "C:\\Program Files (x86)\\BraveSoftware\\Brave-Browser\\Application\\brave.exe";
```

Comment:
not good practice to put static paths in code, this may change by user and not supporting Linux.
i recoomand to have dedicated AgentPAram for it and set it as default for the 64 bit path

Reply:
'BrowserExecutablePath' is a new generic param added for all web agents, Ginger will automatically detect the brave path using this static path, in case it not available on mentioned path, user can provide his custom browser path in new BrowserExecutablePath param.